packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SafetyChecker.sol

```diff
@@ -32,7 +85,7 @@ contract OVM_SafetyChecker is iOVM_SafetyChecker {
         bytes memory _bytecode
     )
         override
-        external
+        public
```

Comment:
we might actually want to make this `internal` now, to ensure that all checked code is cached.